aptos-move/aptos-memory-usage-tracker/src/lib.rs

```diff
@@ -355,6 +355,23 @@ where
         self.base.charge_unpack(is_generic, args)
     }
 
+    #[inline]
+    fn charge_pack_closure(
+        &mut self,
+        is_generic: bool,
+        args: impl ExactSizeIterator<Item = impl ValueView> + Clone,
+    ) -> PartialVMResult<()> {
+        self.use_heap_memory(args.clone().fold(AbstractValueSize::zero(), |acc, val| {
+            acc + self
+                .vm_gas_params()
+                .misc
+                .abs_val
+                .abstract_stack_size(val, self.feature_version())
```

Comment:
Currently, this uses
```
#[inline]
    fn visit_closure(&mut self, _depth: usize, _len: usize) -> bool {
        // TODO(#15664): introduce a dedicated gas parameter?
        self.size += self.params.struct_;
        true
    }
```

Can we fix this param as well to use a dedicate closure param?